### PR TITLE
1176 add subject chain 689

### PR DESCRIPTION
**Pull request by TobiasNx**
(Substitutes #1182 .)

See #1176 .

### CODE REVIEW

File: src/main/resources/alma/common/subjects.xml [hunk 1]
```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rules xmlns="http://www.culturegraph.org/metamorph">
+
+
+  <entity name="subject[]" flushWith="record">
+    <!-- RWSK Schlagwortfolgen fka: Schlagwortketten 1 - 10 -->
```
Comment (blackwinter) [inline src/main/resources/alma/common/subjects.xml:6]: *R**SW**K* = *R*egeln für die ***S***chlag***w***ort*k*atalogisierung :)

**Review by dr0i — APPROVED** [review]
This looks  already very good!
We could, maybe, create a macro and call it 10 times with the appropriate field numbers, at least for the sake of going fast we may just go on.

**Comment by dr0i** [comment]
(The build fails for play's dependency at https://repo.typesafe.com/typesafe/ivy-releases/com.typesafe.activator/activator-launcher/1.3.10/ivys/ivy.xml is not resolvable atm. We can ignore that.)

File: src/main/resources/alma/common/subjects.xml (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/common/subjects.xml]: Thanks.

Comment (blackwinter) [reply src/main/resources/alma/common/subjects.xml]: There are more of these... (`maps/rwsk-indicator.xml` and therein)

Comment (TobiasNx) [reply src/main/resources/alma/common/subjects.xml]: Thanks, I thought I had all before I pushed at first. Will have another look.

Comment (TobiasNx) [reply src/main/resources/alma/common/subjects.xml]: Should be fixed now.

**Comment by dr0i** [comment]
Deployed, see e.g. http://alma.lobid.org/resources/search?q=990171142550206441+AND+subject.componentList.label%3A"Functional+Requirements+for+Bibliographic+Records%22

**Comment by dr0i** [comment]
Merged, closed.

**Pull request merged**